.github/workflows/helm-tests.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   HELM_VERSION: v3.15.2
-  PGO_VERSION: 5.5.2
+  PGO_VERSION: 5.7.0
```

Comment:
I would suggest to align with `5.7.4` which we are using currently. This will also make it easier not to miss when we upgrade again.